libs/cardano-ledger-pretty/cardano-ledger-pretty.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-pretty
-version:            1.3.3.0
+version:            1.3.4.0
```

Sorry, I should have been more clear. You didn't need to update the version, just the changelog.

Version `1.3.3.0` has not been released yet. I think it would be useful if you checked out our release process: https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md
It will explain why this is the case
```suggestion
version:            1.3.3.0
```